akd/src/directory.rs

```diff
@@ -681,10 +681,12 @@ mod tests {
     async fn test_empty_tree_root_hash() -> Result<(), AkdError> {
         let db = AsyncInMemoryDatabase::new();
         let vrf = HardCodedAkdVRF {};
-        let akd = Directory::<_, _>::new::<Blake3>(&db, &vrf, false).await?;
+        let akd = Directory::<_, _>::new::<Blake3_256<BaseElement>>(&db, &vrf, false).await?;
```

Comment:
nit: Why? `Blake3` is much more concise

Reply:
This particular test has a hard-coded hash to check against. I had in mind that a future PR could replace the concise `Blake3` with `TestHash` (for e.g.), which could depend on a feature or be a simple change of variable somewhere to try different hash functions. This test will fail, if that is not taken care of by actually fixing the hash fn to `Blake3_256<BaseElement>`.